dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -198,6 +197,7 @@ internal class OkHttpUploader : Uploader {
                 }
             }
         }
+        response.close()
```

Comment:
Generally speaking, this can lead to the exception according to the description here https://github.com/square/okhttp/blob/4984568367caaf359b82c452bd28b5e192824d1c/okhttp/src/main/kotlin/okhttp3/Response.kt#L295-L303, so it is better to call `close()` on the body itself.